Samples/Util/ConsoleHelper.cs

```diff
@@ -41,17 +42,20 @@ public static void AllowAsciEscapeCodes()
                 Console.ReadKey();
                 return;
             }
-        }
-
+        }
+#endif
+
         /// <summary>
         /// For whatever reason - writing perfectly fine bytes out to the console using c# is overly difficult.
         /// This method will convert the bytes to output, read lines from that string and write the strings out
         /// line by line. Why _this_ seems to be necessary is still unknown but this does seem to work.
         /// </summary>
         /// <param name="utf8bytes"></param>
         public static void OutputResponseToConsole(byte[] utf8bytes)
-        {
-            AllowAsciEscapeCodes();
+        {
+#if !MACOS && !LINUX

```

Comment:
these should be using `RuntimeInformation.IsOSPlatform(OSPlatform.OSX)` and `RuntimeInformation.IsOSPlatform(OSPlatform.Linux)`

Reply:
done